lib/sprockets/commoner.rb

```diff
@@ -10,8 +11,10 @@ def self.sprockets4?
     end
   end
 
+  register_mime_type "application/jsx", extensions: [".js.jsx",".jsx"], charset: :unicode
   register_postprocessor 'application/javascript', ::Sprockets::Commoner::Processor
   register_transformer 'application/json', 'application/javascript', ::Sprockets::Commoner::JSONProcessor
+  register_transformer 'application/jsx', 'application/javascript', ::Sprockets::Commoner::JSXProcessor
```

Comment:
I think just adding the `.jsx` extension as an acceptable `application/javascript` extension is better